src/doc/rustc/src/platform-support.md

```diff
@@ -253,6 +253,7 @@ target | std | host | notes
 `i686-uwp-windows-gnu` | ? |  |
 `i686-uwp-windows-msvc` | ? |  |
 `i686-wrs-vxworks` | ? |  |
+[`loongarch64-unknown-linux-gnuf64`](platform-support/loongarch-linux.md) | ✓ |  | LoongArch64 linux (lp64d ABI)
```

Comment:
I think this should be a `?` for now right? It doesn't look to me like the standard library is currently supported.